data/templates/nginx/server.tpl.conf

```diff
@@ -93,3 +93,84 @@ server {
     access_log /var/log/nginx/{{ domain }}-access.log;
     error_log /var/log/nginx/{{ domain }}-error.log;
 }
+
+# vhost dedicated to XMPP http_upload
+server {
+    listen 443 ssl http2;
+    listen [::]:443 ssl http2;
+    server_name xmpp-upload.{{ domain }};
+
+    location /upload {
+        alias /var/www/xmpp-upload.{{ domain }}/upload;
```

Comment:
Is this path generated by Metronome? Is metronome storing the uploaded files in there?